sycl/test/matrix/matrix-int8-test.cpp

```diff
@@ -1,5 +1,10 @@
-// RUN: %clangxx -fsycl -O2 %s -o %t.out
-// XFAIL: *
+// RUN: %clangxx -fsycl -fsycl-device-only -O2 -S -emit-llvm -o - %s | FileCheck %s
+// RUNx: %clangxx -fsycl -O2 %s -o %t.out
+
+// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL.16" = type { [12 x [12 x [1 x [4 x i32]]]] addrspace(4)* }
+// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL" = type { [12 x [48 x [1 x [4 x i8]]]] addrspace(4)* }
+// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL.14" = type { [48 x [12 x [4 x [4 x i8]]]] addrspace(4)* }
```

Comment:
I suggest replacing `.16` and `.14` in type name with regex matching any numbers.